spec/api-power-monitor-spec.js

```diff
@@ -27,9 +31,7 @@ xdescribe('powerMonitor', () => {
       reset = Promise.promisify(logindMock.Reset, {context: logindMock})
     })
 
-    after(async () => {
-      await reset()
-    })
+    after(async () => { await reset() })
```

Comment:
Just pass `reset` to the `after()` call.
`reset` returns a `Promise`, and `after` can will properly handle it.

Reply:
done